core/keys/symbol_key.talon-list

```diff
@@ -0,0 +1,56 @@
+# symbol_key is for key names that should be available in command mode, but NOT during dictation.
+list: user.symbol_key
+-
+dot: .
+point: .
+quote: "'"
+question: ?
+apostrophe: "'"
+L square: [
+left square: [
+brack: [
+bracket: [
+left bracket: [
+square: [
+R square: ]
```

Comment:
Would recommend you capitalize R everywhere (my preference) or nowhere; right now it's inconsistent.

Reply:
Fair enough, this was auto converted and apparently inconsistent in the original. Fixed